sdcm/sct_config.py

```diff
@@ -95,6 +95,9 @@ class SCTConfiguration(dict):
         dict(name="n_db_nodes", env="SCT_N_DB_NODES",  type=int_or_list,
              help="Number list of database nodes in multiple data centers."),
 
+        dict(name="n_gemini_test_oracle_db_nodes", env="SCT_n_gemini_test_oracle_db_nodes",  type=int_or_list,
```

Comment:
Shouldn't it be all capital?
SCT_N_GEMINI_TEST...

Reply:
yep, you right. Fixed

Reply:
`SCT_N_TEST_ORACLE_DB_NODES`

Reply:
We can also remove gemini from the name.